subsys/fs/littlefs_fs.c

```diff
@@ -627,8 +679,20 @@ static int littlefs_mount(struct fs_mount_t *mountp)
 	lfs_size_t block_size = lcp->block_size;
 
 	if (block_size == 0) {
-		block_size = get_block_size(fs->area);
+		if (littlefs_on_blkdev(mountp)) {
+#ifdef CONFIG_FS_LITTLEFS_BLK_DEV
```

Comment:
Can be merged with `IS_ENABLED(...)`

Reply:
I will replace the `#ifdef CONFIG_FS_LITTLEFS_BKL_DEV` with `#if IS_ENABLED()`